rafthttp/transport.go

```diff
@@ -375,6 +382,11 @@ func (t *Transport) Resume() {
 	}
 }
 
+// InitialPeerNotify returns a channel that closes when an initial
+// peer connection has been established. Use this to wait until the
+// first peer connection becomes active.
+func (t *Transport) InitialPeerNotify() <-chan struct{} { return t.initPeerNotifyCh }
```

Comment:
well... actually, an easy way to solve this problem is to have a for loop to loop over the peer status and check if active is true.

```
for p := range peers {
    if p.status.isActive() {
        // send to chan
    }
}
```